Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizer.cs

```diff
@@ -124,8 +123,7 @@ private bool IsBoundarySystemEnabled()
                 return false;
             }
 
-            MixedRealityConfigurationProfile profile = MixedRealityManager.Instance?.ActiveProfile;
-            return (bool)(profile?.EnableBoundarySystem);
+            return (bool)(MixedRealityManager.Instance?.ActiveProfile?.EnableBoundarySystem);
```

Comment:
as bool?

Reply:
nope. compiler is not happy with that. :)